cli/waiter/subcommands/init.py

```diff
@@ -0,0 +1,44 @@
+import json
+import logging
+import os
+
+from waiter import token_post
+from waiter.util import deep_merge
+
+parser = None
+
+
+DEFAULTS = {
+    'cmd': 'your command goes here',
+    'version': 'your version goes here',
+    'cpus': 0.1,
+    'mem': 2048
+}
+
+
+def init_token_json(_, args, __):
+    """Creates (or updates) a Waiter token"""
+    logging.debug('args: %s' % args)
+    file = os.path.abspath(args.pop('file'))
```

Comment:
We should have a warning (and a force option) when you are overwriting an existing file.

Reply:
Done